install.sh

```diff
@@ -8,6 +8,41 @@ then
     exit 0
 fi
 
+continue=unset
+
+for file in "/usr/share/pulseaudio/alsa-mixer/profile-sets/apple-t2.conf" \
+            "/usr/share/alsa-card-profile/mixer/profile-sets/apple-t2.conf" \
+            "/usr/lib/udev/rules.d/91-pulseaudio-custom.rules" \
+            "/usr/share/alsa/cards/AppleT2.conf"
+do
+    if [ -f "$file" ]
+    then
+        if [ "$continue" = "unset" ]
+        then
+            read -p "Old configuration files are present, do you want to delete them? (y/n) " continue
+
+            if [ "$continue" != "y" ]
+            then
+                break
+            fi
+        fi
+
+        rm -v $file
+    fi
+done
+
+config_home=${XDG_CONFIG_HOME:-~/.config}
+
+for file in "$config_home/pulse/default.pa" \
+            "$config_home/pulse/daemon.conf"
+do
+    if [ -f "$file" ]
+    then
+        echo "You seem to have pulseaudio configuration files present, don't forget to update them if they contain anything specific to T2 Macs"
```

Comment:
Well I guess it shall be better to ask the user to delete them or not?

Reply:
In my opinion the user should review the files to see if they can be deleted as they might have put other settings there and have forgotten about it.